jwql/shared_tasks/shared_tasks.py

```diff
@@ -742,11 +739,10 @@ def run_pipeline(input_file, in_ext, ext_or_exts, instrument, jump_pipe=False):
         uncal_name = os.path.basename(uncal_file)
         result = start_pipeline(uncal_name, short_name, ext_or_exts, instrument, jump_pipe=jump_pipe)
         logging.info("\t\tStarting with ID {}".format(result.id))
-        processed_path = result.get()
```

Comment:
I'm pretty sure we need to keep this. Maybe we can change "processed_path" to "_" if it will make Ruff happier, but I'm pretty sure we need to call the get method.

Reply:
You can also just get rid of `processed_path = ` unless ruff demands an assignment